src/test/java/codeu/controller/SearchServletTest.java

```diff
@@ -0,0 +1,59 @@
+package codeu.controller;
```

Comment:
A trick to keep the PR purely for the bot changes, when the search servlet tests haven't been merged yet is to change the base branch of the PR, from master to the branch of the search tests. This means you're now building on top of that branch instead of master. Note that this means that if you merge the base branch first into master, you'll need to change this PR's base branch back to master to continue diffing against the right set of changes, and to actually merge into master when done.

If that's too confusing it's fine, you don't have to do it. Github doesn't unfortunately handle multiple PRs building on top of each other very well. :)

Reply:
Thanks for this tip!